.appveyor.yml

```diff
@@ -7,4 +7,4 @@ install:
   - cmd: C:\Python36-x64\python -m pip install jinja2 nose pycodestyle
   
 build_script:
-  - cmd: python scripts\build.py -t --python-path "C:\\Python36-x64\\python" --disable_java --require_dotnet --clean
+  - cmd: python scripts\build.py -t --python-path "C:\\Python36-x64\\python" --disable_java --require_dotnet --clean --nobazel
```

Comment:
Does this change result in --nobazel being on by default (I think it does?). In which case, why does the .appveyor.yml need to change?